packages/flutter_tools/lib/src/macos/swift_package_manager.dart

```diff
@@ -10,6 +10,8 @@ import '../plugins.dart';
 import '../project.dart';
 import 'swift_packages.dart';
 
+const String kFlutterGeneratedPluginSwiftPackageName = 'FlutterGeneratedPluginSwiftPackage';
```

Comment:
Consider adding a quick comment explaining what this is. Something like "The Swift package generated by the Flutter tool to inject Swift packages into the Flutter iOS app's build"